samples/extauthz/ext-authz.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: ext-authz
+  labels:
+    app: ext-authz
+spec:
+  ports:
+    - name: http
+      port: 8000
+      targetPort: 8000
+    - name: grpc
+      port: 9000
+      targetPort: 9000
+  selector:
+    app: ext-authz
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ext-authz
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ext-authz
+  template:
+    metadata:
+      labels:
+        app: ext-authz
+      annotations:
+        sidecar.istio.io/componentLogLevel: "rbac:debug,jwt:debug,filter:debug"
```

Comment:
does this work with and without injected sidecar?

Reply:
if not, we should either add inject=false label, or a small comment in readme to make sure you have it in a namespace with injection

Reply:
you mean the sample server? It works in both cases. The annotation is just to make it a bit easier to debug when using with sidecar, and should just be a no-op without sidecar

Reply:
sorry the comment was on the annotation but I meant in general. sounds good